RadarSDK/RadarAPIClient.m

```diff
@@ -911,11 +926,14 @@ - (void)syncLogs:(NSArray<RadarLog *> *)logs completionHandler:(RadarSyncLogsAPI
     NSArray *logsArray = [RadarLog arrayForLogs:logs];
     [params setValue:logsArray forKey:@"logs"];
 
+    // "logPayload = false" enforces "don't log the logging call".
+    // Otherwise, all log entries would continue coalesce and would never fully clear from the buffer
```

Comment:
super nits: "continue to coalesce"

also, we should probably enforce period or no period for both lines

Reply:
fixed